galpy/potential_src/Potential.py

```diff
@@ -1510,6 +1510,166 @@ def nemo_accpars(self,vo,ro):
+        INPUT:
+            
+            Pot - Potential instance or list of such instances
+            
+            R - Galactocentric radius (can be Quantity)
+            
+            z - height (can be Quantity)
+            
+            M - (default = None) Mass of object (can be Quantity)
+            
+        OUTPUT:
+            
+            Tidal Radius
+        
+        HISTORY:
+            
+            2018-02-12 - Written - Webb (UofT)
+            
+        """
+        if self.isNonAxi:
+            #Make sure potential is axisymmetric
+            print("Error - cannot calculate rtide for non-axisymmetric potential")
+            return 0.0
+        elif M==None:
+            #If units not given, assume galpy! GM units?
+            print("Error - mass parameter has not been set")
+            return 0.0
+        else:
+            #Check mass units (default: GALPY units)
+            if M_units=='mass':
+                M/=bovy_conversion.mass_in_msol(self._vo,self._ro)
+            elif M_units=='GM':
+                M/=(bovy_conversion.mass_in_msol(self._vo,self._ro)*bovy_conversion._G)
+            # Parse M if it has units
+            elif _APY_LOADED and isinstance(M,units.Quantity):
+                # Try a bunch of possible units
+                unitFound= False
+                # mass
+                try:
+                    M= M.to(units.Msun).value\
+                        /bovy_conversion.mass_in_msol(self._vo,self._ro)
+                except units.UnitConversionError: pass
+                else:
+                    unitFound= True
+                    if not M_units == 'mass':
+                        raise units.UnitConversionError('M= parameter of %s should have units of %s, but has units of mass instead' % (type(self).__name__,M_units))
+                if not unitFound:
+                    # G x mass
+                    try:
+                        M= M.to(units.pc*units.km**2/units.s**2)\
+                            .value\
+                            /bovy_conversion.mass_in_msol(self._vo,self._ro)\
+                            /bovy_conversion._G
+                    except units.UnitConversionError: pass
+                    else:
+                        unitFound= True
+                        if not M_units == 'mass':
+                            raise units.UnitConversionError('M= parameter of %s should have units of %s, but has units of G x mass instead' % (type(self).__name__,M_units))
+                                    
+            #To calculate omegac in spherical coordinates we need the first derivative of the potential with respect to r
+            r=math.sqrt(R**2.0+z**2.0)
+            theta=nu.arctan2(R, z)
+            
+            Rderiv=-1.0*self.Rforce(R,z,use_physical=False)
+            zderiv=-1.0*self.zforce(R,z,use_physical=False)
+
+            omegac2=(Rderiv*(R/r)+zderiv*(z/r))/r
```

Comment:
``Potential`` instances have a method ``rforce`` that gives the radial force (minus the derivative of the potential wrt spherical radius), isn't that just what you want to use here?